quantum/nodes/classical-register/classical-register.js

```diff
@@ -1,18 +1,20 @@
 module.exports = function(RED) {
   'use strict';
-
+  let classicalRegister = null;
   function ClassicalRegisterNode(config) {
     // Creating node with properties and context
+    classicalRegister = this;
```

Comment:
Very happy with everything else but here you could use the node variable defined a few lines below which correspond to the same object as classicalRegister here